### ISSUE TITLE
Only the game page should not be user-scalable.

### ISSUE THREAD

**Issue by Sothatsit** [Sothatsit/RoyalUrClient#130]
Currently all pages are marked as not user-scalable. This is an accessibility problem, and should be fixed.

**Comment by Sothatsit** [comment]
This has now been fixed!
